### PR TITLE
EthereumSwapProvider: refund

### PR DESCRIPTION
**Pull request by monokh**
### Description

Implement refund method for ethereum swap

### CODE REVIEW

File: src/providers/ethereum/EthereumSwapProvider.js [hunk 1]
```diff
@@ -83,16 +83,13 @@ export default class EthereumSwapProvider extends Provider {
 
   async claimSwap (initiationTxHash, recipientAddress, refundAddress, secret, expiration) {
     const initiationTransaction = await this.getMethod('getTransactionReceipt')(initiationTxHash)
-    const data = this.getRedeemSwapData(secret)
+    const data = padHexStart(secret, 64)
     return this.getMethod('sendTransaction')(initiationTransaction.contractAddress, 0, data)
   }
 
-  getRedeemSwapData (secret) {
```
Comment (monokh) [inline src/providers/ethereum/EthereumSwapProvider.js:90]: found these methods pretty unnecessary

Comment (matthewjablack) [reply src/providers/ethereum/EthereumSwapProvider.js:90]: yep, good call removing them

File: src/providers/ethereum/EthereumSwapProvider.js [hunk 2]
```diff
@@ -83,16 +83,13 @@ export default class EthereumSwapProvider extends Provider {
...
-    return padHexStart(secret, 64)
-  }
-
-  getRefundSwapData () {
-    return ''
+  async refundSwap (initiationTxHash, recipientAddress, refundAddress, secretHash, expiration) {
```
Comment (matthewjablack) [inline src/providers/ethereum/EthereumSwapProvider.js:90]: Should this be standardized as `secretHash` as an argument or `secret` as an argument?

**Review by matthewjablack — APPROVED** [review]
LGTM 🎉  just one small comment

File: src/providers/ethereum/EthereumSwapProvider.js (hunk 2 above)
Comment (monokh) [reply src/providers/ethereum/EthereumSwapProvider.js:90]: secrethash i believe as this is what the contract is created with. If secret was part of the interface, a party not knowing the secret couldn't get a refund which doesn't seem right

**Pull request merged**